src/deadline/client/cli/_groups/bundle_group.py

```diff
@@ -87,6 +87,11 @@ def validate_parameters(ctx, param, value):
     is_flag=True,
     help="Skip any confirmation prompts",
 )
+@click.option(
+    "--save-manifests",
```

Comment:
If we add a `--save-to-history` option here, the combination of that and `--save-manifests` seems redundant and potentially confusing. I think you mentioned that you looked at adding a flag like that, would it be very much extra scope to do that instead of `--save-manifests` as a new option?

Reply:
I think if we want to keep something like `--save-to-history` out of scope, I would remove `--save-manifests` and see about adding `--save-to-history` as a follow-up PR sooner rather than later.

Reply:
We chatted offline and decided to remove saving manifests for the CLI flow for now so we don't bloat the CLI interface